clients/geth/specular/rollup/services/sequencer/sequencer.go

```diff
@@ -249,10 +249,10 @@ func (s *Sequencer) sequencingLoop(genesisRoot common.Hash) {
 			confirmedAssertion.CumulativeGasUsed,
 		)
 		if errors.Is(err, core.ErrInsufficientFunds) {
-			log.Crit("[Sequencer: sequencingLoop] Insufficient Funds to send Tx", "error", err)
+			customlog.Crit("[Sequencer: sequencingLoop] Insufficient Funds to send Tx", "error", err)
```

Comment:
we can get rid of these tags now `[Sequencer: sequencingLoop] `

Reply:
Yes, this is updated